### ISSUE TITLE
Virtual Cloud Desktop Using Apache Guacamole Proposed Changes

### ISSUE THREAD

**Issue by KillerKelvUK** [linode/docs#3019]
Link: https://linode.com/docs/applications/remote-desktop/remote-desktop-using-apache-guacamole-on-docker/
## Issue
The ' ' marks encasing the environment variables are infact passed through to the containers and become characters in the string.  Example command from your guide below showing this for the db name, username and password.

docker run --name example-guacamole --link example-guacd:guacd --link example-mysql:mysql -e MYSQL_DATABASE='guacamole_db' -e MYSQL_USER='guacamole_user' -e MYSQL_PASSWORD='guacamole_user_password' -d -p 127.0.0.1:8080:8080 guacamole/guacamole

The resulting variables are then stored with the ' ' marks e.g. MYSQL_PASSWORD='password'.  Unless the MySQL user account is specifically created with matching set of ' ' marks (which would be odd) this causes the configuration to fail.

## Suggested Fix
Remove the ' ' marks.

**Comment by Rajakavitha1** [comment]
Hi @KillerKelvUK 

We tried to create the password without ' ' 
_mysql> ALTER USER 'root'@'localhost' IDENTIFIED BY *********;_
we get an error that reads:
_ERROR 1064 (42000): You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '*********' at line 1_

Hence marking this issue as closed. Please feel free to reopen the issue if you think any changes need to be made.

**Comment by KillerKelvUK** [comment]
Hi, agree the mysql> commands require the ' ' marks but the docker commands where the problem lies.

**Comment by Rajakavitha1** [comment]
Hi @KillerKelvUK ,

Thanks for the clarification, we will be removing the ' ' shortly and post the link of the preview shortly.

**Comment by Rajakavitha1** [comment]
Hi @KillerKelvUK 

We validated the guide and updated the guide and also incorporated the changes you had suggested. Here's the preview of the updated guide :https://deploy-preview-3047--nostalgic-ptolemy-b01ab8.netlify.com/applications/remote-desktop/remote-desktop-using-apache-guacamole-on-docker/#guacamole-in-browser . The updated guide will be published in the website during our next docs release cycle after the PR is merged.

Closing this issue now.
